cmdstanpy/stanfit.py

```diff
@@ -1635,3 +1917,27 @@ def from_csv(
             'An error occured processing the CSV files:\n\t{}'.format(str(e))
         ) from e
     return fit
+
+
+def add_var_to_xarray(
```

Comment:
I don't think this is the best name for what this function does. Perhaps something like `build_xarray_data`?

Reply:
agree - renamed